code/WorkInProgress/Electronics.dm

```diff
@@ -466,92 +462,273 @@
 
 	var/processing = 0
 	var/net_id = null
-	var/frequency = 1149
+	var/frequency = 1467
 	var/datum/radio_frequency/radio_connection
 	var/no_print_spam = 1 // In relation to world.time.
 	var/olde = 0
+	var/datum/mechanic_controller/ruck_controls
+	var/host_ruck
+	///net_id of the ruck that will send messages
+	var/list/known_rucks = null
+	///list of rucks we've seen send a SYNC or SYNCREPLY (or even DROP but that's weird)
+	var/boot_time = null
```

Comment:
are these  documented upside down?

Reply:
They are, I'm away from the computer right now